src/server/controllers/DeviceController.ts

```diff
@@ -1,32 +1,29 @@
-/* eslint-disable @typescript-eslint/no-misused-promises */
 import { asObject, asString } from 'cleaners'
-import express from 'express'
+import { Serverlet } from 'serverlet'
 
 import { Device } from '../../models'
+import { ApiRequest } from '../../types/request-types'
+import { jsonResponse } from '../../types/response-types'
 
-export const DeviceController = express.Router()
+export const postDevice: Serverlet<ApiRequest> = async request => {
```

Comment:
Seems like some error handling is missing. If fail to register device, shouldn't it return status 500?

Reply:
Right, we are relying on the logging middleware to handle errors for now. This will spam Slack until we get the right `catch` lines in place, but cleaning up these routes will be another PR.